cartography/intel/gcp/bigtable.py

```diff
@@ -0,0 +1,427 @@
+            request = bigtable.projects().instances().tables().list(parent=f"projects/{project_id}/instances/{instance['name']}")
+            while request is not None:
+                response = request.execute()
+                if response.get('tables', []):
+                    for table in response['tables']:
+                        table['instance_id'] = instance['id']
+                        table['id'] = table['name']
+                        bigtable_tables.append(table)
+                request = bigtable.projects().instances().tables().list_next(previous_request=request, previous_response=response)
+            return bigtable_tables
+        except HttpError as e:
+            err = json.loads(e.content.decode('utf-8'))['error']
+            if err.get('status', '') == 'PERMISSION_DENIED' or err.get('message', '') == 'Forbidden':
+                logger.warning(
+                    (
+                        "Could not retrieve Bigtable Instance Tables on project %s due to permissions issues. Code: %s, Message: %s"
+                    ), project_id, err['code'], err['message'],
+                )
+                return []
+            else:
+                raise
+
+
+@timeit
+def load_bigtable_instances(session: neo4j.Session, data_list: List[Dict[str, Optional[str]]], project_id: str, update_tag: int) -> None:
+    session.write_transaction(_load_bigtable_instances_tx, data_list, project_id, update_tag)
+
+
+@timeit
+def _load_bigtable_instances_tx(tx: neo4j.Transaction, bigtable_instances: List[Dict], project_id: str, gcp_update_tag: int) -> None:
+    """
+        :type neo4j_transaction: Neo4j transaction object
+        :param neo4j transaction: The Neo4j transaction object
+
+        :type instances_resp: List
+        :param instances_resp: A list of Bigtable Instances
+
+        :type project_id: str
+        :param project_id: Current Google Project Id
+
+        :type gcp_update_tag: timestamp
+        :param gcp_update_tag: The timestamp value to set our new Neo4j nodes with
+    """
+    ingest_bigtable_instances = """
+    UNWIND {bigtable_instances} as instance
+    MERGE (i:GCPBigtableInstance{id:{instance.id}})
+    ON CREATE SET
+        i.firstseen = timestamp()
+    SET
+        i.name = instance.name,
+        i.displayName = instance.displayName,
+        i.state = instance.state,
+        i.type = instance.type,
+        i.createTime = instance.createTime
+    WITH instance, i
+    MATCH (owner:GCPProject{id:{ProjectId}})
+    MERGE (owner)-[r:RESOURCE]->(i)
+    ON CREATE SET
+        r.firstseen = timestamp(),
+        r.lastupdated = {gcp_update_tag}
+    """
+    tx.run(
+        ingest_bigtable_instances,
+        bigtable_instances=bigtable_instances,
+        ProjectId=project_id,
+        gcp_update_tag=gcp_update_tag,
+    )
+
+
+@timeit
+def load_bigtable_clusters(session: neo4j.Session, data_list: List[Dict[str, Optional[str]]], project_id: str, update_tag: int) -> None:
```

Comment:
In all the load functions, please change type for the objects from `List[Dict[str, Optional[str]]]` to `List[Dict]`